tools/lint/lint.py

```diff
@@ -747,8 +747,19 @@ def changed_files(wpt_root):
 
 def lint_paths(kwargs, wpt_root):
     if kwargs.get("paths"):
-        r = os.path.realpath(wpt_root)
-        paths = [os.path.relpath(os.path.realpath(x), r) for x in kwargs["paths"]]
+        #make new list in case there are multiple added directories
+        AllPaths=[]
+        for path in kwargs.get("paths"):
+            # create extension for called directory and merge it with the existing path
+            extension= path+'/'
+            newPath=(str(os.path.realpath(wpt_root))+'/'+ extension)
```

Comment:
So I feel like this is basically
`abs_path = os.path.realpath(os.path.join(wpt_root, path))`?